projects/storefrontapp-e2e-cypress/cypress/integration/regression/customer-ticketing/ticket-listing.e2e-spec.ts

```diff
@@ -148,6 +167,31 @@ describe('ticket listing', () => {
         cy.go('back');
         customerTicketing.verifyCreatedTicketDetails(ticketToSort);
       });
+
+      it('should take you to the corresponding page when clicking the page number on pagination', () => {
+        customerTicketing.loginAsAdminLindaWolf();
+        customerTicketing.visitElectronicTicketListingPage();
+        customerTicketing.verifyPaginationExist();
+        customerTicketing.selectSortBy(customerTicketing.TestSortingTypes.id);
```

Comment:
nit: Could you please rename customerTicketing.TestSortingTypes.id to customerTicketing.TestSortingTypes.byId